internal/database/actions.go

```diff
@@ -17,10 +17,41 @@ import (
 	"github.com/upper/db/v4"
 )
 
+var (
+	errInvalidDatabasePath = "database: passed an empty database path"
```

Comment:
Do we need to introduce this new error? (If the answer is yes, then it's more idiomatic to do

```Go
var errInvalidDatabasePath = errors.New(/* string */)
```

than what you are doing above.